cypress/e2e/metachain/metachain.spec.ts

```diff
@@ -21,6 +21,7 @@ viewports.forEach((viewport) => {
         "Unrivaled innovation and interoperability, without compromising on scalability or security. Meta Chain is for building dApps on the world's largest blockchains."
       );
     });
+
```

Comment:
can we please add next e2e scenarios:

- check title of the entire FAQ section
- check titles of the each FAQ section
- check desc after expanding of each section
- check that link to the documentation is the correct one

Thank you!